kolibri/core/assets/src/utils/intl-locale-data.js

```diff
@@ -142,8 +142,8 @@ module.exports = function(locale) {
       });
     case 'zh-hans':
       return new Promise(function(resolve) {
-        require.ensure(['intl/locale-data/jsonp/zh-Hans.js'], function(require) {
```

Comment:
I know that have the correct casing for `Hans` was important in the the python context to match it in [this code](https://github.com/django/django/tree/master/django/contrib/auth/locale).

Hopefully it doesn't matter because here it's the name of a JS file, but it would be nicer for it to be consistent to help in grepping this language code.

Reply:
As it is now, it fails compiling the `kolibri.core.default_frontend bundle`, same as the first time we included Chinese, until the file is manually renamed back to `zh-Hans.js`.

![Selection_522](https://user-images.githubusercontent.com/1457929/83294434-98236900-a1ed-11ea-95d0-c1a2a0ed80ee.png)